src/main/java/seedu/address/model/policy/Policy.java

```diff
@@ -0,0 +1,111 @@
+package seedu.address.model.policy;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * The type Policy.
+ */
+public class Policy {
+    /**
+     * The constant MESSAGE_CONSTRAINTS_NAME.
+     */
+    public static final String MESSAGE_CONSTRAINTS_NAME =
+            "Policy names should only contain alphanumeric characters and spaces, and it should not be blank";
+    /**
+     * The constant MESSAGE_CONSTRAINTS_ID.
+     */
+    public static final String MESSAGE_CONSTRAINTS_ID =
+            "Policy ID should only contain numbers, and it should be at least 1 digits long";
+    /**
+     * The constant VALIDATION_NAME_REGEX.
+     */
+    public static final String VALIDATION_NAME_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
+    /**
+     * The constant VALIDATION_ID_REGEX.
+     */
+    public static final String VALIDATION_ID_REGEX = "\\d+";
+    /**
+     * The Policy name.
+     */
+    public final String policyName;
+    /**
+     * The Policy id.
+     */
+    public final String policyID;
+    private final PolicyType type;
+
```

Comment:
When we expand the commands to include PolicyTypes, we can consider if users can choose the policytypes by

1)Numbers -> 1 = Default, 2 = Travel etc
2)Spelling the policytypes out as per enums
3)Shortforms

If we are to use shortform or spelling, we should think if we should allow small/block letters only or are mixed casing allowed like dEfAuLt.